arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -2561,6 +2738,17 @@
       },
       "discriminator": "objectType"
     },
+    "AzureBackupServerEngine": {
+      "description": "Backup engine type when Azure Backup Server is used to manage the backups.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupEngineBase"
+        }
+      ],
+      "properties": {},
```

Comment:
Hi, I fixed the "properties" : {} elements from the document. Please review / merge!